build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
```

Comment:
Dumb question:  Why aren't we updating Version Tool's `BuildInfo.Get` to handle `file://`?